src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { getMovie } from '../../api';
+import { ResponseError } from '../../types/ReponseError';
+import { convertDataIntoMovie } from '../Convertation';
+import { MovieCard } from '../MovieCard/MovieCard';
+
+interface Props {
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [isMovieLoading, setIsMovieLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [result, setResult] = useState<MovieData | null>(null);
+  const [errorValue, setErrorValue] = useState('');
+
+  const sendQuery = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setIsMovieLoading(true);
+
+    getMovie(query).then((response: MovieData | ResponseError) => {
+      if ('Title' in response) {
+        setResult(response);
+      } else {
+        setError(true);
+        setErrorValue(response.Error);
+      }
+    })
+      .finally(() => setIsMovieLoading(false));
+  };
+
+  const reset = () => {
+    setQuery('');
+    setResult(null);
+  };
+
+  const isDisabled = query.trim();
```

Comment:
isDisabled should be a boolean value, use `Boolean(query.trim())` or logical opeartors `!!query.trim()`.